src/components/PreviewComment/index.js

```diff
@@ -44,7 +44,7 @@ const PreviewComment = ({comment, time, image, totalComment, onPress, user, item
           <View style={styles.profile}>
             {item?.data?.anon_user_info_emoji_name || item?.data?.is_anonymous ? (
               <View style={[styles.image, {backgroundColor: item.data.anon_user_info_color_code}]}>
-                <Text style={{color: COLORS.white, fontSize: 14}}>
+                <Text style={{color: 'white', fontSize: 14}}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The hardcoded color 'white' could be replaced with a constant for better maintainability. If the color needs to be changed in the future, it would be easier to do so from a single location.
```diff
- <Text style={{color: 'white', fontSize: 14}}>
+ <Text style={{color: colors.white, fontSize: 14}}>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->